app.js

```diff
@@ -1,8 +1,84 @@
-// import functions
+import { getPokedex } from './fetch.js';
 
+// import functions
+console.log('Its ALIVE!');
 // grab DOM elements
+const template = document.querySelector('#template');
+const list = document.querySelector('#list');
+const audio = document.querySelector('#audio');
+const response = document.querySelector('#response');
+const input = document.querySelector('#Answer');
+const button = document.querySelector('#button');
 
+console.log('test', template, list, audio, response, input);
 // set event listeners 
     // get user input
     // use user input to update state 
     // update DOM to reflect the new state
+
+input.addEventListener('submit', () => {
+
+    const inputValue = input.value;
+    console.log(inputValue);
+    
+    if ('Answer' === 'Yes') {
+
+        response.textContent = 'You have good taste.';
+
+    } 
+    else {
+
+        response.textContent = 'Please answer correctly.';
+    
+    }
+
+});
+
+button.addEventListener('click', () => {
+
+ 
+    let answer = input.value;
+    if (answer === 'Yes' || answer === 'yes') {
+        response.textContent = 'You have good taste.';
+    } else {
+        response.textContent = 'Please answer correctly.';
+    }
+});
+
+
+
+
+    
+async function getPokemon() {
+    const pokemons = await getPokedex();
+    for (let pokemon of pokemons) {
+        const clone = template.content.cloneNode(true);
+        const pokemonName = clone.querySelector('#pokemon');
```

Comment:
You'll want to be careful assigning ids to everything in your app. An ID is a property that is absolutely unique and will not be applicable to any other element rendered in your app. Since you are rendering a list of items here, each of those items is going to have the same id associated with them. Typically, you will add a unique reference to the id, such as the name or other property of the object you're rendering from the API. This becomes more applicable when you get to React, though, so just something to keep in mind!